src/style.css

```
@@ -1,75 +1,94 @@
-@import 'https://fonts.googleapis.com/css?family=Roboto:300,400,500';
-body {
-    margin: 0 auto;
-    max-width: 50em;
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+body{
+    background-color: #E6EBEF;
+    
+    width: 100%;
+    height: 100vh;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+#home {
```

Tu devrais pas utiliser d'id dans ton css pour cette page
```suggestion
.card{
```